src/window/window_wrapper/mod.rs

```diff
@@ -232,6 +233,8 @@ pub fn start_loop(
         .with_maximized(SETTINGS.get::<CmdLineSettings>().maximized)
         .with_decorations(!SETTINGS.get::<CmdLineSettings>().frameless);
 
+    let winit_window_builder = winit_window_builder.with_app_id("Neovide".to_string());
```

Comment:
Does this need a cfg to make sure it only runs when on unix machines?

Reply:
Good catch! I haven't tested on any other platform to be honest, and I think this should be true.
Same would be for the `class` and `instance` vars, I'll try to cover them too!

Reply:
After a quick review, I think it might not be enough to check for just 'unix', instead it might need a finer selection between x11 and wayland environments..
I can try to implement it, but no guarantees given 😅